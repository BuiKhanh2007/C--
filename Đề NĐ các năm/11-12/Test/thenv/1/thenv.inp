5
1234567
2345678
3456789
4567890
1357964
