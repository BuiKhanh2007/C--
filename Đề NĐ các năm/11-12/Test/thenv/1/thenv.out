E
D
C
I
D
